### PR TITLE
Replaced some simple assert_object_equals cases.

### PR DESCRIPTION
**Pull request by sideshowbarker**
Part of https://github.com/w3c/web-platform-tests/issues/2033

### CODE REVIEW

**Comment by gsnedders** [comment]
```
15:01 < aryehgregor> Um, I might have done something wrong by mistake.
15:01 < aryehgregor> I think I pushed to the wrong repo?
```

**Comment by ayg** [comment]
I can't figure out how to restore the branch at this point, so please re-push it if you still have it locally in case I don't wind up finding a solution.  (I deleted it by mistake.)

**Comment by sideshowbarker** [comment]
> I can't figure out how to restore the branch at this point, so please re-push it if you still have it locally in case I don't wind up finding a solution. (I deleted it by mistake.)

OK, I just pushed it from clone and it looks like this PR is now back into the correct state again

File: workers/interfaces/DedicatedWorkerGlobalScope/postMessage/structured-clone-message.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
       {test:async_test('regexp'), check:function(e) { assert_equals('' + e.data, '/foo/'); assert_equals(e.data instanceof RegExp, true, 'e.data instanceof RegExp'); }},
       {test:async_test('self'), check:function(e) { assert_equals(e.data, null); }},
       {test:async_test('array'), check:function(e) { assert_array_equals(e.data, [undefined, null, false, true, 1, NaN, Infinity, 'foo', null, null]); }},
-      {test:async_test('object'), check:function(e) { assert_object_equals(e.data, {a:undefined, b:null, c:false, d:true, e:1, f:NaN, g:Infinity, h:'foo', k:null, n:null}); }},
+      {test:async_test('object'), check:function(e) { assert_equals(JSON.stringify(e.data), JSON.stringify({a:undefined, b:null, c:false, d:true, e:1, f:NaN, g:Infinity, h:'foo', k:null, n:null})); }},
```
Comment (annevk) [inline workers/interfaces/DedicatedWorkerGlobalScope/postMessage/structured-clone-message.html]: Does JSON.stringify have a defined order?

Comment (annevk) [inline workers/interfaces/DedicatedWorkerGlobalScope/postMessage/structured-clone-message.html]: Perhaps it's better to just enumerate the stuff we want to assert?

**Comment by foolip** [comment]
@stephenmcgruer here's an open PR in case you missed it in https://github.com/web-platform-tests/wpt/issues/2033.

**Pull request closed without merging**